vanillajs/cards/app.js

```diff
@@ -0,0 +1,25 @@
+const placeholderInfo = {
+  image: "https://via.placeholder.com/150/771796",
+  name: "Jane Doe",
+  title: "VP Engineering"
+};
+
+class Card {
+  constructor(container) {
+    this.container = document.querySelector(container);
+  }
+
+  init() {
+    const card = `
+    <div class="card">
+      <img src=${placeholderInfo.image} alt="placeholder">
```

Comment:
Your component should receive `data`, not placeholderInfo. Also, make the alt tag part of that data.